src/templates/en/index.html

```diff
@@ -69,11 +70,50 @@ <h2>
         </button>
       </div>
       <div class="intro-year">2019</div>
-      <div class="intro-image">
-        <img class="home-browser" src="/static/images/home-browser.png" />
-        <img class="camada-751" src="/static/images/camada-751.png" />
-        <img class="camada-759" src="/static/images/camada-759.png" />
-        <img class="camada-752" src="/static/images/camada-752.png" />
+      <div class="intro-image-container">
+        <div class="intro-image">
+          <img class="home-browser" src="/static/images/home-browser.png" />
+          <img class="camada-751" src="/static/images/camada-751.png" />
+          <img class="camada-759" src="/static/images/camada-759.png" />
+          <img class="camada-752" src="/static/images/camada-752.png" />
+        </div>
+      </div>
+      <div class="menu">
+        <li>
+          <a href="#">Table of Contents</a>
+        </li>
+        <li>
+          <a href="#">Contributors</a>
+        </li>
+        <li>
+          <a href="#">Methodology</a>
+        </li>
+        <div>
+          <img src="/static/images/ha-home.png" />
+          <div class="social-media">
+            <img src="/static/images/twitter.png" />
```

Comment:
Link to https://twitter.com/HTTPArchive

Reply:
For each of these social links, they should be gray/transparent by default and yellow on hover. We should be able to use CSS filters to adjust the color overlay if we're not already using SVGs.

![image](https://user-images.githubusercontent.com/1120896/65060986-f1d95100-d946-11e9-95c1-58c864db478d.png)